main.go

```diff
@@ -19,12 +20,33 @@ func main() {
 		os.Exit(1)
 	}
 
-	packages, err := pathrunner.Walk(os.Args[1])
+	f, err := os.OpenFile(".gammaray.log", os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
Can we override this to allow `os.Stderr` as well via viper configuration? My thought process here is that if we package gammaray in a container we might want to output it to stderr instead of a file.

Reply:
Yes, I wanted to put a proper logger to do that actually (`logrus` seems a good candidate), as I think some log levels would be welcome (e.g. the `vesrionformatter` stuff is quite verbose and rarely useful)

In #15 you can at least define a different file, and defining "/dev/stderr" results in using stderr, though it is not great

I think we should put a logging issue aside fro this PR covering that, do you agree ?

Reply:
Makes perfect sense. And also logrus is my favourite.